src/server/controllers/login.js

```diff
@@ -11,10 +11,8 @@ exports.get = async (req, res) => {
     if (isAuth) {
       res.cookie('jwt', token, { maxAge: 6048000000 });
       res.send(token);
-    } else {
-      res.send({ Error: 'Check Username Or Password' });
-    }
+    } else throw new Error();
   } catch (error) {
-    res.send({ Error: 'There Is Error In Our Network' });
+    res.status(400).send({ Error: 'Check username or password' });
```

Comment:
@knanahassouna1 This change is an improvement because the login endpoint no longer sends a 200 response for username/password errors, but I think the ideal way to respond to errors would be to treat the auth errors separate from all other errors.  I think that would mean res.send-ing a 401 ("check username or password") if isAuth is false, up near line 13, instead of throwing an error, and keeping the res.status(400) in the catch block but changing the error message being sent.
Then in the client code, you could again display the endpoint's error message, or choose to show "Please check username or password" or some other message based on the http status code.

On most endpoints, this wouldn't be as important, but for a login endpoint, it matters more:  Many projects will have a "forgot password" or "reset password" flow if you enter the wrong credentials (or might lock you out of the app if you enter them wrong too many times), and you don't want to send the user down that flow if the error is really just that your DB is temporarily down, or they have an unreliable wifi connection in that moment, or whatever else.